go.mod

```diff
@@ -77,3 +77,5 @@ require (
 	google.golang.org/genproto v0.0.0-20210319143718-93e7006c17a6 // indirect
 	google.golang.org/grpc v1.36.0 // indirect
 )
+
+require github.com/hashicorp/terraform-plugin-log v0.4.1
```

Comment:
I think adding this was not intentional? You can `go mod tidy` it out.

Reply:
Good catch, thanks!